doc/nrf/device_guides/nrf91/nrf91_cloud_certificate.rst

```diff
@@ -0,0 +1,176 @@
+#. Click :guilabel:`Select device` and select the DK from the drop-down list.
+
+   .. tabs::
+
+      .. group-tab:: nRF9161 DK
+
+         .. figure:: images/cellularmonitor_selectdevice_nrf9161.png
+            :alt: Cellular Monitor - Select device
+
+            Cellular Monitor - Select device
+
+      .. group-tab:: nRF9160 DK
+
+         .. figure:: images/cellularmonitor_selectdevice1_nrf9160.png
+            :alt: Cellular Monitor - Select device
+
+            Cellular Monitor - Select device
+
+   The drop-down text changes to the type of the selected device, with the SEGGER ID below the name.
+
+#. Click the :guilabel:`Open Serial Terminal` option of the `Cellular Monitor`_ app to open the Serial Terminal.
+
+   .. tabs::
+
+      .. group-tab:: nRF9161 DK
+
+         .. figure:: images/cellularmonitor_open_serial_terminal_nrf9161.png
+            :alt: Cellular Monitor - Open Serial Terminal
+
+            Cellular Monitor - Open Serial Terminal
+
+      .. group-tab:: nRF9160 DK
+
+         .. figure:: images/cellularmonitor_open_serial_terminal.png
+            :alt: Cellular Monitor - Open Serial Terminal
+
+            Cellular Monitor - Open Serial Terminal
+
+#. Enter ``AT+CFUN=4`` in the text field for AT commands and click :guilabel:`Send`.
+   This AT command puts the modem to offline state.
+#. Enter ``AT+CFUN?`` in the text field for AT commands and click :guilabel:`Send`.
+   This AT command returns the state of the modem.
+
+   The command must return ``+CFUN: 4``, which indicates that the modem is in offline state.
+   If it returns a different value, repeat the previous step.
+#. Open the Cellular Monitor app.
+#. Click :guilabel:`CERTIFICATE MANAGER` in the navigation bar to switch to the certificate manager view.
+
+   .. tabs::
+
+      .. group-tab:: nRF9161 DK
+
+         .. figure:: images/cellularmonitor_navigationcertificatemanager_nrf9161.png
+            :alt: Cellular Monitor - Certificate Manager
+
+            Cellular Monitor - Certificate Manager
+
+      .. group-tab:: nRF9160 DK
+
+         .. figure:: images/cellularmonitor_navigationcertificatemanager.png
+            :alt: Cellular Monitor - Certificate Manager
+
+            Cellular Monitor - Certificate Manager
+
+#. Click :guilabel:`Load from JSON` and select the :file:`*.cert.json` file that you downloaded from nRF Cloud.
+   Alternatively, you can drag and drop the file onto the GUI.
+#. Ensure that the **Security tag** is set to ``16842753``, which is the security tag for nRF Cloud credentials.
+#. Click :guilabel:`Update certificate`.
+
+   The log message "Certificate update completed" indicates that the certificate was provisioned successfully.
+   If you encounter any errors, switch to the terminal view and check the output of the AT commands that were sent to the nRF9160 DK modem.
+
+   .. note::
+
+      If you have connected your nRF9160 DK to nRF Cloud before, you must delete the device there after provisioning the certificate.
```

Comment:
```suggestion
      If you have connected your |DK| to nRF Cloud before, you must delete the device there after provisioning the certificate.
```

Reply:
I have removed this label, as it is not needed any more and replaced |DK| with DK, wherever applicable. Please check.